src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/addonput.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Would be great if you could help sort the files like this in aks-preview 😄

Reply:
Yes, I have the changes in my private branch. I will be creating a PR to update aks-preview too and bring it up to speed with the changes similar to what I have in this PR.

Reply:
https://github.com/Azure/azure-cli-extensions/pull/6241